code/modules/spells/spell_types/personality_commune.dm

```diff
@@ -0,0 +1,35 @@
+/obj/effect/proc_holder/spell/targeted/personality_commune
+	name = "Personality Commune"
+	desc = "Sends thoughts to your alternate consciousness."
+	charge_max = 0
+	clothes_req = FALSE
+	range = -1
+	include_user = TRUE
+	action_icon_state = "telepathy"
+	action_background_icon_state = "bg_spell"
+	var/datum/brain_trauma/severe/split_personality/trauma
+	var/flufftext = "You hear an echoing voice in the back of your head..."
+	var/boldnotice = "boldnotice"
+	var/notice = "notice"
```

Comment:
?

Reply:
That was copied from telepathy, but I'll change it I guess